### PR TITLE
Fix formatting in Mac Step 1 markdown

### PR DESCRIPTION
**Pull request by nusco**

### CODE REVIEW

File: docs/mac/step_one.md [hunk 1]
```diff
@@ -148,32 +148,23 @@ Virtual Box VM, it maintains its configuration between uses.
...
-
-	To try something more ambitious, you can run an Ubuntu container with:
-	 $ docker run -it ubuntu bash
-
-	Share images, automate workflows, and more with a free Docker Hub account:
-	 https://hub.docker.com
-
-	For more examples and ideas, visit:
-	 https://docs.docker.com/userguide/
+	    Unable to find image 'hello-world:latest' locally
```
Comment (thaJeztah) [inline docs/mac/step_one.md]: Hm, you used tabs here to indent, but the existing indentation uses 4 spaces. Can you change your tabs to 4 spaces?

**Comment by thaJeztah** [comment]
looks like this is a duplicate of https://github.com/docker/tutorials/pull/55.

**Comment by nusco** [comment]
Yes, now I see that this is probably be a diff zero from #55. Feel free to close this if you merge that one.
(BTW, I signed off and replaced tabs with spaces, just in case).

**Comment by thaJeztah** [comment]
#55 has been merged, so I'll close this


Thanks for contributing @nusco !

**Comment by nusco** [comment]
You're welcome :smiley:

**Pull request closed without merging**